main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -203,7 +202,7 @@ object Main {
           System.exit(1)
       }
     } catch {
-      case ex: FlixError =>
+      case ex: Exception => // TODO JLS: Maybe delete this and call Main.main() instead or keep
```

Comment:
@magnus-madsen Is there any reason for this manual exception throw? HoleError had a nice string given to the super class (FlixError) so when throw normally it looks nice but in this case the toString is indirectly used to it looked wrong

Reply:
We might as well let it bubble then?

Reply:
I think it would be the same except some formatting